challenges/platform/troubleshooting.md

```diff
@@ -1,16 +1,14 @@
 # Summary
 
-This exercise simulates a troubleshooting session where several faults and failures have been created in a sample environment, requiring the candidate to investigate and solve problems. 
+This exercise simulates a troubleshooting session; The candidate must investigate and solve several faults and failures across Linux servers.
```

Comment:
Super nit-picky, but a capitalized word should not follow a semicolon unless the word is a proper noun or an acronym.  :)

Reply:
Thanks!  Fixed in 4efe257.